src/js/api/posts/create.js

```diff
@@ -0,0 +1,30 @@
+import { dummyApiUrl } from "../constants.js";
+import { dummyApiCreatePost } from "../constants.js";
+import { authBaseFetchOpen } from "../apiBaseFetch.js";
+
+/**
+ * Creates a new listing
+ * @param {string} listData - Represents input information
+ */
+export async function create(listData) {
+  const createListingUrl = dummyApiUrl + dummyApiCreatePost;
+
+  console.log(createListingUrl)
+
+  const data = await authBaseFetchOpen(createListingUrl, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(listData),
+  }
+  );
+
+  if (data.ok) {
+    // return await response.json();
+    const response = await data.json();
```

Comment:
This says that if data is ok, then it should make a const called response.
In other words, it does not return anything.
You could perhaps try: return await data.json()

Reply:
Have made the change.